frontend/src/components/common/Label.tsx

```diff
@@ -2,9 +2,15 @@ import * as React from 'react';
 import './Label.scss';
```

Comment:
This was from alpha branch, will switch over as we handle required differently now